test/features/support/env.rb

```diff
@@ -28,3 +27,9 @@
 World(OllertTest::Navigation) do
   Ollert.new
 end
+
+Trello.configure do |config|
+  config.developer_public_key = ENV['INTEGRATION_KEY']
```

Comment:
I removed the test token altogether and just went with `PUBLIC_KEY`. I also renamed `INTEGRATION_TOKEN` :arrow_right: `MEMBER_TOKEN` to keep with ruby-trello vernacular.